File: algorithm/golang/two_sum.go
package main

import "fmt"

func twoSum(nums []int, target int) []int {
    for i := 0; i < len(nums); i++ {
		for j := i + 1; j < len(nums); j++ {
			if nums[i] + nums[j] == target {
				    return []int{i, j}
			}
		}
	}
    return []int{0, 0}
}

func main() {
    lst := []int{2, 3, 4}
    fmt.Println(twoSum(lst, 6))
}

File: algorithm/javascript/two_sum.js
var twoSum = function(nums, target) {
    for(var i = 0; i < nums.length; i = i + 1){
        for (var j = i + 1 ; j < nums.length ; j = j + 1){
            if (nums[i] + nums[j] == target) {
                return [i, j];
            }
        }
    }
};